159356-1702-Extramural/capstone
# Feature/play knight card

To test:
- Buy some dev cards (currently set to return knights)
- Play the knight card
- Confirm once the first player has played the knight other players can't use the knight
- Confirm robber moves to new location next round, and that location never blocks the player who played the knight
